lib/organisms/gravity-form/gravity-form.controller.js

```diff
@@ -127,21 +127,31 @@ function lnOGravityFormController($log, $scope, $element, $attrs, $sce, lnOGravi
   // Init
   //
 
-  $log.debug('lnOGravityFormController -> ', vm, '/ $element: ', $element);
-
-  if (vm.loadFromAPI === true) {
-    lnOGravityFormService.getForm(vm.id)
-      .then(checkLoadFormResponse, checkLoadFormResponse);
+  if (angular.isUndefined(lnOGravityFormService.getConfig())) {
+    var unbindInit = $scope.$on('lnOGravityFormConfigUpdated', init);
+  } else {
+    init();
   }
 
+  function init() {
+    $log.debug('lnOGravityFormController -> ', vm, '/ $element: ', $element);
 
-  vm.form.id = 'gform_' + vm.id;
+    unbindInit();
```

Comment:
Do we first need to check unbindInit exists? For the case where the app is bootstrapped immediately.